Package.resolved

```diff
@@ -3,10 +3,10 @@
     "pins": [
       {
         "package": "swift-baggage-context",
-        "repositoryURL": "https://github.com/slashmo/gsoc-swift-baggage-context.git",
+        "repositoryURL": "https://github.com/slashmo/gsoc-swift-baggage-context",
         "state": {
           "branch": null,
-          "revision": "349804a0599573d09df6e433cb8266b80727e5b0",
+          "revision": "10469603494f1cde67fbd0725cdde0e04d131b05",
```

Comment:
Could you please update this to the latest version? I think I force-pushed the tag directly after creation, could have messed it up a bit 😬 Also, do you know if there's a difference in having the `.git` suffix. I think I remember some SwiftPM bug where it has to be `.git` everywhere, but could have already been fixed and I'm still paranoid by it 😁

Reply:
Thought that's the latest, will double check with an update 👍